readme.md

```diff
@@ -7,30 +7,28 @@
   <a href="https://discord.gg/3CVqX5bphG" alt="Discord"><img src="https://img.shields.io/discord/829309363213565982?label=Discord&style=flat&logo=discord&logoColor=white"></img></a>
   <a href="https://github.com/raluvy95/jolly/issues" alt="Issues"><img src="https://img.shields.io/github/issues/raluvy95/jolly"></img></a>
 </p>
-A discord bot designed for one server at once (private bot). Rewritten in
-TypeScript using Deno. The predecessor variant of this can be found in
+A discord bot designed for my server (private bot). Rewritten in
+TypeScript using Deno. Its predecessor can be found in
```

Comment:
It's

Reply:
I'm pretty sure it's still "its", but feel free to look it up somewhere idk

Reply:
it's is an abbreviation of it is, making it it's, just like don't and can't being abbreviations of do not and can not